{"log":"I0818 14:04:12.607387       1 glogsink.go:42] {\"verb\":\"ADDED\",\"event\":{\"metadata\":{\"name\":\"ict.169c6b93e20d2d44\",\"namespace\":\"problem-app\",\"selfLink\":\"/api/v1/namespaces/problem-app/events/ict.169c6b93e20d2d44\",\"uid\":\"ef89b615-8321-49f1-b27f-2bd9f19cc8a9\",\"resourceVersion\":\"479028\",\"creationTimestamp\":\"2021-08-18T14:04:08Z\",\"managedFields\":[{\"manager\":\"kubelet\",\"operation\":\"Update\",\"apiVersion\":\"v1\",\"time\":\"2021-08-18T14:04:08Z\"}]},\"involvedObject\":{\"kind\":\"Pod\",\"namespace\":\"problem-app\",\"name\":\"ict\",\"uid\":\"e9b2d046-2f3d-4382-81ec-ed5fb796ed6d\",\"apiVersion\":\"v1\",\"resourceVersion\":\"479013\"},\"reason\":\"FailedCreatePodSandBox\",\"message\":\"Failed to create pod sandbox: rpc error: code = Unknown desc = failed to set up sandbox container \\\"33e4850578a20d5d3016d76502096d74181c1b2ea26988575402ad742c5efbf6\\\" network for pod \\\"ict\\\": networkPlugin cni failed to set up pod \\\"ict_problem-app\\\" network: add cmd: failed to assign an IP address to container\",\"source\":{\"component\":\"kubelet\",\"host\":\"ip-192-168-50-69.ap-northeast-2.compute.internal\"},\"firstTimestamp\":\"2021-08-18T14:04:08Z\",\"lastTimestamp\":\"2021-08-18T14:04:08Z\",\"count\":1,\"type\":\"Warning\",\"eventTime\":null,\"reportingComponent\":\"\",\"reportingInstance\":\"\"}}\n","stream":"stderr","time":"2021-08-18T14:04:12.610016072Z"}
{"log":"I0818 14:04:12.607462       1 glogsink.go:42] {\"verb\":\"ADDED\",\"event\":{\"metadata\":{\"name\":\"ict.169c6b94cbf831e5\",\"namespace\":\"problem-app\",\"selfLink\":\"/api/v1/namespaces/problem-app/events/ict.169c6b94cbf831e5\",\"uid\":\"1cf757f4-9899-45d8-8a22-ff04213cfaf6\",\"resourceVersion\":\"479047\",\"creationTimestamp\":\"2021-08-18T14:04:12Z\",\"managedFields\":[{\"manager\":\"kubelet\",\"operation\":\"Update\",\"apiVersion\":\"v1\",\"time\":\"2021-08-18T14:04:12Z\"}]},\"involvedObject\":{\"kind\":\"Pod\",\"namespace\":\"problem-app\",\"name\":\"ict\",\"uid\":\"e9b2d046-2f3d-4382-81ec-ed5fb796ed6d\",\"apiVersion\":\"v1\",\"resourceVersion\":\"479013\",\"fieldPath\":\"spec.containers{ict}\"},\"reason\":\"Failed\",\"message\":\"Failed to pull image \\\"nginx2\\\": rpc error: code = Unknown desc = Error response from daemon: pull access denied for nginx2, repository does not exist or may require 'docker login': denied: requested access to the resource is denied\",\"source\":{\"component\":\"kubelet\",\"host\":\"ip-192-168-50-69.ap-northeast-2.compute.internal\"},\"firstTimestamp\":\"2021-08-18T14:04:12Z\",\"lastTimestamp\":\"2021-08-18T14:04:12Z\",\"count\":1,\"type\":\"Warning\",\"eventTime\":null,\"reportingComponent\":\"\",\"reportingInstance\":\"\"}}\n","stream":"stderr","time":"2021-08-18T14:04:12.610037558Z"}
{"log":"I0818 14:04:27.618476       1 glogsink.go:42] {\"verb\":\"UPDATED\",\"event\":{\"metadata\":{\"name\":\"ict.169c6b94cbf831e5\",\"namespace\":\"problem-app\",\"selfLink\":\"/api/v1/namespaces/problem-app/events/ict.169c6b94cbf831e5\",\"uid\":\"1cf757f4-9899-45d8-8a22-ff04213cfaf6\",\"resourceVersion\":\"479110\",\"creationTimestamp\":\"2021-08-18T14:04:12Z\",\"managedFields\":[{\"manager\":\"kubelet\",\"operation\":\"Update\",\"apiVersion\":\"v1\",\"time\":\"2021-08-18T14:04:12Z\"}]},\"involvedObject\":{\"kind\":\"Pod\",\"namespace\":\"problem-app\",\"name\":\"ict\",\"uid\":\"e9b2d046-2f3d-4382-81ec-ed5fb796ed6d\",\"apiVersion\":\"v1\",\"resourceVersion\":\"479013\",\"fieldPath\":\"spec.containers{ict}\"},\"reason\":\"Failed\",\"message\":\"Failed to pull image \\\"nginx2\\\": rpc error: code = Unknown desc = Error response from daemon: pull access denied for nginx2, repository does not exist or may require 'docker login': denied: requested access to the resource is denied\",\"source\":{\"component\":\"kubelet\",\"host\":\"ip-192-168-50-69.ap-northeast-2.compute.internal\"},\"firstTimestamp\":\"2021-08-18T14:04:12Z\",\"lastTimestamp\":\"2021-08-18T14:04:27Z\",\"count\":2,\"type\":\"Warning\",\"eventTime\":null,\"reportingComponent\":\"\",\"reportingInstance\":\"\"},\"old_event\":{\"metadata\":{\"name\":\"ict.169c6b94cbf831e5\",\"namespace\":\"problem-app\",\"selfLink\":\"/api/v1/namespaces/problem-app/events/ict.169c6b94cbf831e5\",\"uid\":\"1cf757f4-9899-45d8-8a22-ff04213cfaf6\",\"resourceVersion\":\"479047\",\"creationTimestamp\":\"2021-08-18T14:04:12Z\",\"managedFields\":[{\"manager\":\"kubelet\",\"operation\":\"Update\",\"apiVersion\":\"v1\",\"time\":\"2021-08-18T14:04:12Z\"}]},\"involvedObject\":{\"kind\":\"Pod\",\"namespace\":\"problem-app\",\"name\":\"ict\",\"uid\":\"e9b2d046-2f3d-4382-81ec-ed5fb796ed6d\",\"apiVersion\":\"v1\",\"resourceVersion\":\"479013\",\"fieldPath\":\"spec.containers{ict}\"},\"reason\":\"Failed\",\"message\":\"Failed to pull image \\\"nginx2\\\": rpc error: code = Unknown desc = Error response from daemon: pull access denied for nginx2, repository does not exist or may require 'docker login': denied: requested access to the resource is denied\",\"source\":{\"component\":\"kubelet\",\"host\":\"ip-192-168-50-69.ap-northeast-2.compute.internal\"},\"firstTimestamp\":\"2021-08-18T14:04:12Z\",\"lastTimestamp\":\"2021-08-18T14:04:12Z\",\"count\":1,\"type\":\"Warning\",\"eventTime\":null,\"reportingComponent\":\"\",\"reportingInstance\":\"\"}}\n","stream":"stderr","time":"2021-08-18T14:04:27.618758545Z"}
{"log":"I0818 14:04:57.610534       1 glogsink.go:42] {\"verb\":\"UPDATED\",\"event\":{\"metadata\":{\"name\":\"ict.169c6b94cbf831e5\",\"namespace\":\"problem-app\",\"selfLink\":\"/api/v1/namespaces/problem-app/events/ict.169c6b94cbf831e5\",\"uid\":\"1cf757f4-9899-45d8-8a22-ff04213cfaf6\",\"resourceVersion\":\"479201\",\"creationTimestamp\":\"2021-08-18T14:04:12Z\",\"managedFields\":[{\"manager\":\"kubelet\",\"operation\":\"Update\",\"apiVersion\":\"v1\",\"time\":\"2021-08-18T14:04:12Z\"}]},\"involvedObject\":{\"kind\":\"Pod\",\"namespace\":\"problem-app\",\"name\":\"ict\",\"uid\":\"e9b2d046-2f3d-4382-81ec-ed5fb796ed6d\",\"apiVersion\":\"v1\",\"resourceVersion\":\"479013\",\"fieldPath\":\"spec.containers{ict}\"},\"reason\":\"Failed\",\"message\":\"Failed to pull image \\\"nginx2\\\": rpc error: code = Unknown desc = Error response from daemon: pull access denied for nginx2, repository does not exist or may require 'docker login': denied: requested access to the resource is denied\",\"source\":{\"component\":\"kubelet\",\"host\":\"ip-192-168-50-69.ap-northeast-2.compute.internal\"},\"firstTimestamp\":\"2021-08-18T14:04:12Z\",\"lastTimestamp\":\"2021-08-18T14:04:57Z\",\"count\":3,\"type\":\"Warning\",\"eventTime\":null,\"reportingComponent\":\"\",\"reportingInstance\":\"\"},\"old_event\":{\"metadata\":{\"name\":\"ict.169c6b94cbf831e5\",\"namespace\":\"problem-app\",\"selfLink\":\"/api/v1/namespaces/problem-app/events/ict.169c6b94cbf831e5\",\"uid\":\"1cf757f4-9899-45d8-8a22-ff04213cfaf6\",\"resourceVersion\":\"479110\",\"creationTimestamp\":\"2021-08-18T14:04:12Z\",\"managedFields\":[{\"manager\":\"kubelet\",\"operation\":\"Update\",\"apiVersion\":\"v1\",\"time\":\"2021-08-18T14:04:12Z\"}]},\"involvedObject\":{\"kind\":\"Pod\",\"namespace\":\"problem-app\",\"name\":\"ict\",\"uid\":\"e9b2d046-2f3d-4382-81ec-ed5fb796ed6d\",\"apiVersion\":\"v1\",\"resourceVersion\":\"479013\",\"fieldPath\":\"spec.containers{ict}\"},\"reason\":\"Failed\",\"message\":\"Failed to pull image \\\"nginx2\\\": rpc error: code = Unknown desc = Error response from daemon: pull access denied for nginx2, repository does not exist or may require 'docker login': denied: requested access to the resource is denied\",\"source\":{\"component\":\"kubelet\",\"host\":\"ip-192-168-50-69.ap-northeast-2.compute.internal\"},\"firstTimestamp\":\"2021-08-18T14:04:12Z\",\"lastTimestamp\":\"2021-08-18T14:04:27Z\",\"count\":2,\"type\":\"Warning\",\"eventTime\":null,\"reportingComponent\":\"\",\"reportingInstance\":\"\"}}\n","stream":"stderr","time":"2021-08-18T14:04:57.610716542Z"}
{"log":"I0818 14:05:49.617074       1 glogsink.go:42] {\"verb\":\"UPDATED\",\"event\":{\"metadata\":{\"name\":\"ict.169c6b94cbf831e5\",\"namespace\":\"problem-app\",\"selfLink\":\"/api/v1/namespaces/problem-app/events/ict.169c6b94cbf831e5\",\"uid\":\"1cf757f4-9899-45d8-8a22-ff04213cfaf6\",\"resourceVersion\":\"479357\",\"creationTimestamp\":\"2021-08-18T14:04:12Z\",\"managedFields\":[{\"manager\":\"kubelet\",\"operation\":\"Update\",\"apiVersion\":\"v1\",\"time\":\"2021-08-18T14:04:12Z\"}]},\"involvedObject\":{\"kind\":\"Pod\",\"namespace\":\"problem-app\",\"name\":\"ict\",\"uid\":\"e9b2d046-2f3d-4382-81ec-ed5fb796ed6d\",\"apiVersion\":\"v1\",\"resourceVersion\":\"479013\",\"fieldPath\":\"spec.containers{ict}\"},\"reason\":\"Failed\",\"message\":\"Failed to pull image \\\"nginx2\\\": rpc error: code = Unknown desc = Error response from daemon: pull access denied for nginx2, repository does not exist or may require 'docker login': denied: requested access to the resource is denied\",\"source\":{\"component\":\"kubelet\",\"host\":\"ip-192-168-50-69.ap-northeast-2.compute.internal\"},\"firstTimestamp\":\"2021-08-18T14:04:12Z\",\"lastTimestamp\":\"2021-08-18T14:05:49Z\",\"count\":4,\"type\":\"Warning\",\"eventTime\":null,\"reportingComponent\":\"\",\"reportingInstance\":\"\"},\"old_event\":{\"metadata\":{\"name\":\"ict.169c6b94cbf831e5\",\"namespace\":\"problem-app\",\"selfLink\":\"/api/v1/namespaces/problem-app/events/ict.169c6b94cbf831e5\",\"uid\":\"1cf757f4-9899-45d8-8a22-ff04213cfaf6\",\"resourceVersion\":\"479201\",\"creationTimestamp\":\"2021-08-18T14:04:12Z\",\"managedFields\":[{\"manager\":\"kubelet\",\"operation\":\"Update\",\"apiVersion\":\"v1\",\"time\":\"2021-08-18T14:04:12Z\"}]},\"involvedObject\":{\"kind\":\"Pod\",\"namespace\":\"problem-app\",\"name\":\"ict\",\"uid\":\"e9b2d046-2f3d-4382-81ec-ed5fb796ed6d\",\"apiVersion\":\"v1\",\"resourceVersion\":\"479013\",\"fieldPath\":\"spec.containers{ict}\"},\"reason\":\"Failed\",\"message\":\"Failed to pull image \\\"nginx2\\\": rpc error: code = Unknown desc = Error response from daemon: pull access denied for nginx2, repository does not exist or may require 'docker login': denied: requested access to the resource is denied\",\"source\":{\"component\":\"kubelet\",\"host\":\"ip-192-168-50-69.ap-northeast-2.compute.internal\"},\"firstTimestamp\":\"2021-08-18T14:04:12Z\",\"lastTimestamp\":\"2021-08-18T14:04:57Z\",\"count\":3,\"type\":\"Warning\",\"eventTime\":null,\"reportingComponent\":\"\",\"reportingInstance\":\"\"}}\n","stream":"stderr","time":"2021-08-18T14:05:49.617214203Z"}
{"log":"I0818 14:34:12.609441       1 glogsink.go:42] {\"verb\":\"UPDATED\",\"event\":{\"metadata\":{\"name\":\"ict.169c6b94cbf831e5\",\"namespace\":\"problem-app\",\"selfLink\":\"/api/v1/namespaces/problem-app/events/ict.169c6b94cbf831e5\",\"uid\":\"1cf757f4-9899-45d8-8a22-ff04213cfaf6\",\"resourceVersion\":\"479357\",\"creationTimestamp\":\"2021-08-18T14:04:12Z\",\"managedFields\":[{\"manager\":\"kubelet\",\"operation\":\"Update\",\"apiVersion\":\"v1\",\"time\":\"2021-08-18T14:04:12Z\"}]},\"involvedObject\":{\"kind\":\"Pod\",\"namespace\":\"problem-app\",\"name\":\"ict\",\"uid\":\"e9b2d046-2f3d-4382-81ec-ed5fb796ed6d\",\"apiVersion\":\"v1\",\"resourceVersion\":\"479013\",\"fieldPath\":\"spec.containers{ict}\"},\"reason\":\"Failed\",\"message\":\"Failed to pull image \\\"nginx2\\\": rpc error: code = Unknown desc = Error response from daemon: pull access denied for nginx2, repository does not exist or may require 'docker login': denied: requested access to the resource is denied\",\"source\":{\"component\":\"kubelet\",\"host\":\"ip-192-168-50-69.ap-northeast-2.compute.internal\"},\"firstTimestamp\":\"2021-08-18T14:04:12Z\",\"lastTimestamp\":\"2021-08-18T14:05:49Z\",\"count\":4,\"type\":\"Warning\",\"eventTime\":null,\"reportingComponent\":\"\",\"reportingInstance\":\"\"},\"old_event\":{\"metadata\":{\"name\":\"ict.169c6b94cbf831e5\",\"namespace\":\"problem-app\",\"selfLink\":\"/api/v1/namespaces/problem-app/events/ict.169c6b94cbf831e5\",\"uid\":\"1cf757f4-9899-45d8-8a22-ff04213cfaf6\",\"resourceVersion\":\"479357\",\"creationTimestamp\":\"2021-08-18T14:04:12Z\",\"managedFields\":[{\"manager\":\"kubelet\",\"operation\":\"Update\",\"apiVersion\":\"v1\",\"time\":\"2021-08-18T14:04:12Z\"}]},\"involvedObject\":{\"kind\":\"Pod\",\"namespace\":\"problem-app\",\"name\":\"ict\",\"uid\":\"e9b2d046-2f3d-4382-81ec-ed5fb796ed6d\",\"apiVersion\":\"v1\",\"resourceVersion\":\"479013\",\"fieldPath\":\"spec.containers{ict}\"},\"reason\":\"Failed\",\"message\":\"Failed to pull image \\\"nginx2\\\": rpc error: code = Unknown desc = Error response from daemon: pull access denied for nginx2, repository does not exist or may require 'docker login': denied: requested access to the resource is denied\",\"source\":{\"component\":\"kubelet\",\"host\":\"ip-192-168-50-69.ap-northeast-2.compute.internal\"},\"firstTimestamp\":\"2021-08-18T14:04:12Z\",\"lastTimestamp\":\"2021-08-18T14:05:49Z\",\"count\":4,\"type\":\"Warning\",\"eventTime\":null,\"reportingComponent\":\"\",\"reportingInstance\":\"\"}}\n","stream":"stderr","time":"2021-08-18T14:34:12.609517172Z"}
{"log":"I0818 14:34:12.611354       1 glogsink.go:42] {\"verb\":\"UPDATED\",\"event\":{\"metadata\":{\"name\":\"ict.169c6b93e20d2d44\",\"namespace\":\"problem-app\",\"selfLink\":\"/api/v1/namespaces/problem-app/events/ict.169c6b93e20d2d44\",\"uid\":\"ef89b615-8321-49f1-b27f-2bd9f19cc8a9\",\"resourceVersion\":\"479028\",\"creationTimestamp\":\"2021-08-18T14:04:08Z\",\"managedFields\":[{\"manager\":\"kubelet\",\"operation\":\"Update\",\"apiVersion\":\"v1\",\"time\":\"2021-08-18T14:04:08Z\"}]},\"involvedObject\":{\"kind\":\"Pod\",\"namespace\":\"problem-app\",\"name\":\"ict\",\"uid\":\"e9b2d046-2f3d-4382-81ec-ed5fb796ed6d\",\"apiVersion\":\"v1\",\"resourceVersion\":\"479013\"},\"reason\":\"FailedCreatePodSandBox\",\"message\":\"Failed to create pod sandbox: rpc error: code = Unknown desc = failed to set up sandbox container \\\"33e4850578a20d5d3016d76502096d74181c1b2ea26988575402ad742c5efbf6\\\" network for pod \\\"ict\\\": networkPlugin cni failed to set up pod \\\"ict_problem-app\\\" network: add cmd: failed to assign an IP address to container\",\"source\":{\"component\":\"kubelet\",\"host\":\"ip-192-168-50-69.ap-northeast-2.compute.internal\"},\"firstTimestamp\":\"2021-08-18T14:04:08Z\",\"lastTimestamp\":\"2021-08-18T14:04:08Z\",\"count\":1,\"type\":\"Warning\",\"eventTime\":null,\"reportingComponent\":\"\",\"reportingInstance\":\"\"},\"old_event\":{\"metadata\":{\"name\":\"ict.169c6b93e20d2d44\",\"namespace\":\"problem-app\",\"selfLink\":\"/api/v1/namespaces/problem-app/events/ict.169c6b93e20d2d44\",\"uid\":\"ef89b615-8321-49f1-b27f-2bd9f19cc8a9\",\"resourceVersion\":\"479028\",\"creationTimestamp\":\"2021-08-18T14:04:08Z\",\"managedFields\":[{\"manager\":\"kubelet\",\"operation\":\"Update\",\"apiVersion\":\"v1\",\"time\":\"2021-08-18T14:04:08Z\"}]},\"involvedObject\":{\"kind\":\"Pod\",\"namespace\":\"problem-app\",\"name\":\"ict\",\"uid\":\"e9b2d046-2f3d-4382-81ec-ed5fb796ed6d\",\"apiVersion\":\"v1\",\"resourceVersion\":\"479013\"},\"reason\":\"FailedCreatePodSandBox\",\"message\":\"Failed to create pod sandbox: rpc error: code = Unknown desc = failed to set up sandbox container \\\"33e4850578a20d5d3016d76502096d74181c1b2ea26988575402ad742c5efbf6\\\" network for pod \\\"ict\\\": networkPlugin cni failed to set up pod \\\"ict_problem-app\\\" network: add cmd: failed to assign an IP address to container\",\"source\":{\"component\":\"kubelet\",\"host\":\"ip-192-168-50-69.ap-northeast-2.compute.internal\"},\"firstTimestamp\":\"2021-08-18T14:04:08Z\",\"lastTimestamp\":\"2021-08-18T14:04:08Z\",\"count\":1,\"type\":\"Warning\",\"eventTime\":null,\"reportingComponent\":\"\",\"reportingInstance\":\"\"}}\n","stream":"stderr","time":"2021-08-18T14:34:12.611906965Z"}
{"log":"I0818 15:04:12.609611       1 glogsink.go:42] {\"verb\":\"UPDATED\",\"event\":{\"metadata\":{\"name\":\"ict.169c6b94cbf831e5\",\"namespace\":\"problem-app\",\"selfLink\":\"/api/v1/namespaces/problem-app/events/ict.169c6b94cbf831e5\",\"uid\":\"1cf757f4-9899-45d8-8a22-ff04213cfaf6\",\"resourceVersion\":\"479357\",\"creationTimestamp\":\"2021-08-18T14:04:12Z\",\"managedFields\":[{\"manager\":\"kubelet\",\"operation\":\"Update\",\"apiVersion\":\"v1\",\"time\":\"2021-08-18T14:04:12Z\"}]},\"involvedObject\":{\"kind\":\"Pod\",\"namespace\":\"problem-app\",\"name\":\"ict\",\"uid\":\"e9b2d046-2f3d-4382-81ec-ed5fb796ed6d\",\"apiVersion\":\"v1\",\"resourceVersion\":\"479013\",\"fieldPath\":\"spec.containers{ict}\"},\"reason\":\"Failed\",\"message\":\"Failed to pull image \\\"nginx2\\\": rpc error: code = Unknown desc = Error response from daemon: pull access denied for nginx2, repository does not exist or may require 'docker login': denied: requested access to the resource is denied\",\"source\":{\"component\":\"kubelet\",\"host\":\"ip-192-168-50-69.ap-northeast-2.compute.internal\"},\"firstTimestamp\":\"2021-08-18T14:04:12Z\",\"lastTimestamp\":\"2021-08-18T14:05:49Z\",\"count\":4,\"type\":\"Warning\",\"eventTime\":null,\"reportingComponent\":\"\",\"reportingInstance\":\"\"},\"old_event\":{\"metadata\":{\"name\":\"ict.169c6b94cbf831e5\",\"namespace\":\"problem-app\",\"selfLink\":\"/api/v1/namespaces/problem-app/events/ict.169c6b94cbf831e5\",\"uid\":\"1cf757f4-9899-45d8-8a22-ff04213cfaf6\",\"resourceVersion\":\"479357\",\"creationTimestamp\":\"2021-08-18T14:04:12Z\",\"managedFields\":[{\"manager\":\"kubelet\",\"operation\":\"Update\",\"apiVersion\":\"v1\",\"time\":\"2021-08-18T14:04:12Z\"}]},\"involvedObject\":{\"kind\":\"Pod\",\"namespace\":\"problem-app\",\"name\":\"ict\",\"uid\":\"e9b2d046-2f3d-4382-81ec-ed5fb796ed6d\",\"apiVersion\":\"v1\",\"resourceVersion\":\"479013\",\"fieldPath\":\"spec.containers{ict}\"},\"reason\":\"Failed\",\"message\":\"Failed to pull image \\\"nginx2\\\": rpc error: code = Unknown desc = Error response from daemon: pull access denied for nginx2, repository does not exist or may require 'docker login': denied: requested access to the resource is denied\",\"source\":{\"component\":\"kubelet\",\"host\":\"ip-192-168-50-69.ap-northeast-2.compute.internal\"},\"firstTimestamp\":\"2021-08-18T14:04:12Z\",\"lastTimestamp\":\"2021-08-18T14:05:49Z\",\"count\":4,\"type\":\"Warning\",\"eventTime\":null,\"reportingComponent\":\"\",\"reportingInstance\":\"\"}}\n","stream":"stderr","time":"2021-08-18T15:04:12.60969497Z"}
{"log":"I0818 15:04:12.611804       1 glogsink.go:42] {\"verb\":\"UPDATED\",\"event\":{\"metadata\":{\"name\":\"ict.169c6b93e20d2d44\",\"namespace\":\"problem-app\",\"selfLink\":\"/api/v1/namespaces/problem-app/events/ict.169c6b93e20d2d44\",\"uid\":\"ef89b615-8321-49f1-b27f-2bd9f19cc8a9\",\"resourceVersion\":\"479028\",\"creationTimestamp\":\"2021-08-18T14:04:08Z\",\"managedFields\":[{\"manager\":\"kubelet\",\"operation\":\"Update\",\"apiVersion\":\"v1\",\"time\":\"2021-08-18T14:04:08Z\"}]},\"involvedObject\":{\"kind\":\"Pod\",\"namespace\":\"problem-app\",\"name\":\"ict\",\"uid\":\"e9b2d046-2f3d-4382-81ec-ed5fb796ed6d\",\"apiVersion\":\"v1\",\"resourceVersion\":\"479013\"},\"reason\":\"FailedCreatePodSandBox\",\"message\":\"Failed to create pod sandbox: rpc error: code = Unknown desc = failed to set up sandbox container \\\"33e4850578a20d5d3016d76502096d74181c1b2ea26988575402ad742c5efbf6\\\" network for pod \\\"ict\\\": networkPlugin cni failed to set up pod \\\"ict_problem-app\\\" network: add cmd: failed to assign an IP address to container\",\"source\":{\"component\":\"kubelet\",\"host\":\"ip-192-168-50-69.ap-northeast-2.compute.internal\"},\"firstTimestamp\":\"2021-08-18T14:04:08Z\",\"lastTimestamp\":\"2021-08-18T14:04:08Z\",\"count\":1,\"type\":\"Warning\",\"eventTime\":null,\"reportingComponent\":\"\",\"reportingInstance\":\"\"},\"old_event\":{\"metadata\":{\"name\":\"ict.169c6b93e20d2d44\",\"namespace\":\"problem-app\",\"selfLink\":\"/api/v1/namespaces/problem-app/events/ict.169c6b93e20d2d44\",\"uid\":\"ef89b615-8321-49f1-b27f-2bd9f19cc8a9\",\"resourceVersion\":\"479028\",\"creationTimestamp\":\"2021-08-18T14:04:08Z\",\"managedFields\":[{\"manager\":\"kubelet\",\"operation\":\"Update\",\"apiVersion\":\"v1\",\"time\":\"2021-08-18T14:04:08Z\"}]},\"involvedObject\":{\"kind\":\"Pod\",\"namespace\":\"problem-app\",\"name\":\"ict\",\"uid\":\"e9b2d046-2f3d-4382-81ec-ed5fb796ed6d\",\"apiVersion\":\"v1\",\"resourceVersion\":\"479013\"},\"reason\":\"FailedCreatePodSandBox\",\"message\":\"Failed to create pod sandbox: rpc error: code = Unknown desc = failed to set up sandbox container \\\"33e4850578a20d5d3016d76502096d74181c1b2ea26988575402ad742c5efbf6\\\" network for pod \\\"ict\\\": networkPlugin cni failed to set up pod \\\"ict_problem-app\\\" network: add cmd: failed to assign an IP address to container\",\"source\":{\"component\":\"kubelet\",\"host\":\"ip-192-168-50-69.ap-northeast-2.compute.internal\"},\"firstTimestamp\":\"2021-08-18T14:04:08Z\",\"lastTimestamp\":\"2021-08-18T14:04:08Z\",\"count\":1,\"type\":\"Warning\",\"eventTime\":null,\"reportingComponent\":\"\",\"reportingInstance\":\"\"}}\n","stream":"stderr","time":"2021-08-18T15:04:12.611930034Z"}